cmd/carpenter/command.go

```diff
@@ -66,6 +162,8 @@ func run(args arguments) error {
 		io.Filenames = args.files
 	}
 
+	fmt.Println("processing files", io.Filenames)
```

Comment:
Remove debug line and/or move it into the `if len(args.files)` check

Reply:
Ah right lemme remove this